jiant/modules/simple_modules.py

```diff
@@ -234,3 +234,30 @@ def forward(self, s1, s2, mask1, mask2, idx1=[], idx2=[]):
         pair_emb = torch.cat([emb1, emb2, torch.abs(emb1 - emb2), emb1 * emb2], 1)
         logits = self.classifier(pair_emb)
         return logits
+
+
+class TokenProjectionEncoder(nn.Module):
```

Comment:
Just checking my understanding: `TokenProjectionEncoder` applies the same linear projections to each vec in a sequence of vectors, and `TokenMultiProjectionEncoder` applies multiple linear projections to each vec (like multi-head attention) ?

Are you normally setting `d_proj << d_inp` ? They're the same by default

Reply:
After taking a look, I've refactored this to make it based on a dict of classifiers.

Reply:
Actually, decided to go with removed the unnecessary projection layer. Pushed update.